contrib/inventory/vmware_inventory.py

```diff
@@ -279,6 +284,9 @@ def read_settings(self):
 
         # behavior control
         self.maxlevel = int(config.get('vmware', 'max_object_level'))
+        # maxlevel >= 4 required for 'use_hw_if_as_host' to work
+        if config.get('vmware', 'use_hw_if_as_host').lower() == 'true' and self.maxlevel < 4:
```

Comment:
I would prefer you just raise an error if `use_hw_if_as_host` is `true` and maxlevel < 4.